frontend/src/components/common/Terminal.tsx

```diff
@@ -278,17 +287,28 @@ export default function Terminal(props: TerminalProps) {
       fitAddonRef.current = new FitAddon();
       xtermRef.current.xterm.loadAddon(fitAddonRef.current);
 
-      const command = getCurrentShellCommand();
-
-      xtermRef.current.xterm.writeln(t('Trying to run "{{command}}"…', { command }) + '\n');
-
       (async function () {
-        execRef.current = await item.exec(
-          container,
-          (items: ArrayBuffer) => onData(xtermRef.current!, items),
-          { command: [command], failCb: () => shellConnectFailed(xtermRef.current!) }
-        );
-
+        if (isAttach) {
+          xtermRef?.current?.xterm.writeln(
+            t(`Trying to attach to the container {{ container }}...`, { container }) + '\n'
```

Comment:
Shouldn't be a string literal as it'll result in warnings when running make i18n.

Reply:
updated